Header header

int32 img_height
int32 img_width

int32 x
int32 y
